src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_ISymbolInitializer.vb

```diff
@@ -0,0 +1,241 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.Semantics
+Imports Microsoft.CodeAnalysis.Test.Utilities
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+Imports Roslyn.Test.Utilities
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests.Semantics
+
+    Partial Public Class IOperationTests
+        Inherits BasicTestBase
+
+        <Fact, WorkItem(17595, "https://github.com/dotnet/roslyn/issues/17595")>
+        Public Sub NoInitializers()
+            Dim source = <compilation>
+                             <file name="c.vb">
+                                 <![CDATA[
+Class C
+	Shared s1 As Integer
+	Private i1 As Integer
+End Class
+]]>
+                             </file>
+                         </compilation>
+
+            Dim compilation = CreateCompilationWithMscorlib(source, options:=TestOptions.ReleaseDll, parseOptions:=TestOptions.Regular)
+
+            Dim tree = compilation.SyntaxTrees.Single()
+            Dim nodes = tree.GetRoot().DescendantNodes().OfType(Of VariableDeclaratorSyntax)().ToArray()
+            Assert.Equal(2, nodes.Length)
+
+            Dim semanticModel = compilation.GetSemanticModel(tree)
+            For Each node In nodes
+                Assert.Null(semanticModel.GetOperationInternal(node))
+            Next
+        End Sub
+
+        <Fact, WorkItem(17595, "https://github.com/dotnet/roslyn/issues/17595")>
+        Public Sub ConstantInitializers()
+            Dim source = <![CDATA[
+Class C
+	Shared s1 As Integer = 1
+	Private i1 As Integer = 1
```

Comment:
Ah, you are correct. I had just added a separate variable declaration to the same field declaration statement, but this doesn't create multiple InitializedFields in the same IFieldInitializer.

I tried to add a new test with the following code: `Dim x, y As New Object` and the return IFieldInitializer returns a single field symbol 'x'. I debugged into the binder and it seem [InitializerSemanticModel.Bind](http://source.roslyn.io/#Microsoft.CodeAnalysis.VisualBasic/Binding/InitializerSemanticModel.vb,108) always uses the single member symbol for create the bound node. This looks like a semantic model bug to me. I will create a tracking bug for it and mark this test as skipped on it.